dvc/fs/dvc.py

```diff
@@ -109,11 +94,6 @@ def __init__(
 
         from pygtrie import Trie
 
-        if repo is None:
-            repo, repo_factory = self._repo_from_fs_config(
-                subrepos=subrepos, **kwargs
-            )
-
```

Comment:
This is dead, but let's keep it as it is useful and is actually needed for making dvcfs public.